schemas/com.snowplowanalytics.mobile/screen_summary/jsonschema/1-0-0

```diff
@@ -0,0 +1,50 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an entity tracked with foreground/background/screen_end events with summary statistics about the screen view",
+	"self": {
+		"vendor": "com.snowplowanalytics.mobile",
+		"name": "screen_summary",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"type": "object",
+	"properties": {
+		"foreground_sec": {
+			"type": "number",
+			"description": "Time in seconds spent on the current screen while the app was in foreground",
+			"minimum": 0,
+			"maximum": 2147483647
+		},
+		"background_sec": {
+			"type": [
+				"number",
+				"null"
+			],
+			"description": "Time in seconds spent on the current screen while the app was in background",
+			"minimum": 0,
+			"maximum": 2147483647
+		},
+		"last_item_index": {
+			"type": [
+				"integer",
+				"null"
+			],
+			"description": "Index of the last viewed item in a list on the screen",
+			"minimum": 0,
+			"maximum": 65535
+		},
+		"items_count": {
+			"type": [
+				"integer",
+				"null"
+			],
+			"description": "Total number of list items on the screen",
```

Comment:
These descriptions confuse me a little, total number of list items, but the index of the last viewed item _in_ a list? Maybe something like 'Total number of items in the list on the screen'? Or can you have multiple lists on one screen?

Reply:
Good point! Have unified the wording, it is expected that there is only one list on the screen.